resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/Utils.scala

```diff
@@ -131,4 +133,41 @@ object Utils extends Logging {
         s"under spark home test dir ${sparkHomeDir.toAbsolutePath}!")
     }
   }
+
+  def getTestFileAbsolutePath(fileName: String, sparkHomeDir: Path): String = {
```

Comment:
@skonto . In this PR, you copied `getExamplesJarAbsolutePath` to `getTestFileAbsolutePath` and removed all instances using `getExamplesJarAbsolutePath`. Do we think we need to maintain the unused function, `getExamplesJarAbsolutePath`?

In this case, I'd like to recommend you to generalize the existing function by renaming.

Reply:
Hmmm let me check.

Reply:
You are right we dont need it anymore.

Reply:
You are right we dont need it anymore, I will remove it.